layers/meta-resin-beaglebone/recipes-core/fix-mmc-bbb/files/fix-mmc-bbb.conf

```diff
@@ -1 +1,3 @@
-vm.vfs_cache_pressure=400
+vm.min_free_kbytes=8192
+vm.dirty_ratio=5
+vm.dirty_background_ratio=10
```

Comment:
The defaults are `vm.dirty_ratio=10` and `vm.dirty_background_ratio=20`. I halved them